File: manager/src/main/java/com/mydemo/manager/service/PositionTypeService.java
package com.mydemo.manager.service;

import com.mydemo.manager.entity.PositionType;

import java.util.List;

public interface PositionTypeService {
    public List<PositionType> findPositionByPid(int pid);
    public PositionType findOne(int id);
    public int updatePositionType(PositionType positionType);
    public int addPositionType(PositionType positionType);
    public int dele(Integer[] ids) throws Exception;
}

File: manager/src/main/resources/templates/js/service/positionTypeService.js
app.service('positionTypeService',function($http){


    this.findByParentId=function (pid) {
        return $http.get('../position/findByPid?pid='+pid);
    }

    this.findOne=function (id) {
        return $http.get('../position/findOne?id='+id);
    }
    this.update=function (entity) {
        return $http.post('../position/updatePositionType',entity);
    }
    this.add=function (entity) {
        return $http.post('../position/addPositionType',entity);
    }
    this.dele=function (ids) {
        return $http.get('../position/dele?ids='+ids);
    }
});